### PR TITLE
Umcs 33/change/pay later

### PR DESCRIPTION
**Pull request by Ryouzanpaku**
- replace guaranteed payment types by secured payment types.

### CODE REVIEW

File: examples/index.php [hunk 1]
```diff
@@ -204,28 +204,10 @@ function printMessage($type, $title, $text)
...
-                </div>
-                <div class="card olive">
-                    <div class="content">
-                        <div class="header">
-                            Invoice Factoring
-                        </div>
-                        <div class="description">
-                            This example adds the necessary customer data within the php controller.
-                            Please refer to the example of <i>Invoice guaranteed</i> if you want to add the customer data withing the payment form.
+                            Unzer Invoice (secured)
```
Comment (sixer1182) [inline examples/index.php:207]:
```suggestion
                            Unzer Invoice

File: README.md [hunk 2]
```diff
@@ -17,21 +17,20 @@ Please refer to the following documentation for installation instructions and us
 *   [Debugging](https://docs.unzer.com/docs/logging-and-debugging)
 
 ## Supported payment types
+*   Alipay
 *   Card (credit card and debit card) + Recurring
+*   EPS
 *   Giropay
 *   iDEAL
-*   Invoice (guaranteed)
 *   PayPal + Recurring
```
Comment (sixer1182) [inline README.md:25]:
```suggestion
*   Invoice
*   PayPal + Recurring
```

File: examples/index.php [hunk 3]
```diff
@@ -270,24 +252,24 @@ function printMessage($type, $title, $text)
                 <div class="card olive">
                     <div class="content">
                         <div class="header">
-                            SEPA direct debit guaranteed
+                            Unzer Direct Debit (secured)
```
Comment (sixer1182) [inline examples/index.php:255]:
```suggestion
                            Unzer Direct Debit
```

File: README.md [hunk 4]
```diff
@@ -17,21 +17,20 @@ Please refer to the following documentation for installation instructions and us
...
 *   Prepayment
 *   Przelewy24
-*   SEPA direct debit (guaranteed) + Recurring
 *   SOFORT
```
Comment (sixer1182) [inline README.md:28]:
```suggestion
*   SEPA direct debit + Recurring
*   SOFORT
```

File: src/Constants/ApiResponseCodes.php [hunk 5]
```diff
@@ -45,8 +45,8 @@ class ApiResponseCodes
     public const API_ERROR_ORDER_ID_ALREADY_IN_USE                     = 'API.320.200.138';
     public const API_ERROR_RESOURCE_DOES_NOT_BELONG_TO_MERCHANT        = 'API.320.200.145';
     public const API_ERROR_CHARGED_AMOUNT_HIGHER_THAN_EXPECTED         = 'API.330.100.007';
-    public const API_ERROR_IVF_REQUIRES_CUSTOMER                       = 'API.330.100.008';
-    public const API_ERROR_IVF_REQUIRES_BASKET                         = 'API.330.100.023';
+    public const API_ERROR_FACTORING_REQUIRES_CUSTOMER                 = 'API.330.100.008';
```
Comment (sixer1182) [inline src/Constants/ApiResponseCodes.php:48]: check changes in migration guide and changelog

File: src/Constants/ApiResponseCodes.php [hunk 6]
```diff
@@ -45,8 +45,8 @@ class ApiResponseCodes
...
+    public const API_ERROR_FACTORING_REQUIRES_BASKET                   = 'API.330.100.023';
```
Comment (sixer1182) [inline src/Constants/ApiResponseCodes.php:49]: see above

**Review by sixer1182 — CHANGES_REQUESTED** [review]

File: src/Constants/ApiResponseCodes.php (hunk 5 above)
Comment (Ryouzanpaku) [reply src/Constants/ApiResponseCodes.php:48]: Was actually missing, i updated both.

File: src/Constants/ApiResponseCodes.php (hunk 6 above)
Comment (Ryouzanpaku) [reply src/Constants/ApiResponseCodes.php:49]: see above.

**Review by sixer1182 — APPROVED** [review]
LGTM

**Review by DerAusHeidelberg — APPROVED** [review]

**Pull request merged**